### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adambraimbridge**
I'm totally not married to any of the edits here. It's 100% fine as it is already :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,20 +106,26 @@ render(welcome())
 [buble]: https://github.com/Rich-Harris/buble
 [buble-jsx]: https://buble.surge.sh/guide/#jsx
 
-## Overview
+## Specifications
```
Comment (adambraimbridge) [inline README.md]: I'm not really happy with "Specifications", but it's slightly too detailed for "Overview"

File: README.md [hunk 2]
```diff
@@ -103,20 +106,26 @@ render(welcome())
...
 
 ### Properties
 
-Properties are an object containing [HTML attributes][attrs] and values. Attribute names may be written in camelCase or in lowercase. For example, the attribute `tabindex` may be written as `tabIndex`. Any HTML attributes written in camelCase will be converted to lowercase but they will not be hyphenated. Attributes requiring hyphens, such as `aria-*` and `data-*`, should be written with hyphens.
+Properties are expressed in an object containing [HTML attributes][attrs] and values.
```
Comment (adambraimbridge) [inline README.md]: Is this correct terminology?

File: README.md [hunk 3]
```diff
@@ -168,7 +176,7 @@ Please note that child elements will not be rendered for [void elements][void].
```
Comment (adambraimbridge) [inline README.md]: Another example or two would be super helpful here

File: README.md [hunk 4]
```diff
@@ -179,14 +187,16 @@ Hyperons supports the `dangerouslySetInnerHTML` property to inject unescaped HTM
 
 ```jsx
```
Comment (adambraimbridge) [inline README.md]: Another example or two would be super helpful here, too :)

**Comment by adambraimbridge** [comment]
By the way: I'm excited about Hyperons. I want to rebuild houston.in.ft.com as either a single page react app, or render everything serverside & store the HTML in S3 ...

**Comment by i-like-robots** [comment]
ooh, nice one... readmes are hard, eh!

File: README.md (hunk 2 above)
Comment (i-like-robots) [reply README.md]: defined? declared?

File: README.md [hunk 5]
```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/i-like-robots/hyperons.svg?branch=master)](https://travis-ci.org/i-like-robots/hyperons) [![Coverage Status](https://coveralls.io/repos/github/i-like-robots/hyperons/badge.svg?branch=master)](https://coveralls.io/github/i-like-robots/hyperons) [![npm version](https://badge.fury.io/js/hyperons.svg)](https://badge.fury.io/js/hyperons)
 
-Renders components written in JSX to HTML without a framework, on the server or in the browser.
+### Write code in JSX. Render it to HTML.
```
Comment (i-like-robots) [inline README.md]: maybe just emphasis rather than a heading? Probably good to keep the word "components" in there

**Comment by i-like-robots** [comment]
Let's sit down and get this 🚀

**Comment by i-like-robots** [comment]
Changes included in #16

**Pull request closed without merging**